autoload/OmniSharp.vim

```diff
@@ -517,7 +515,7 @@ endfunction
 
 function! OmniSharp#ServerIsRunning() abort
   try
-    python vim.command("let s:alive = '" + getResponse("/checkalivestatus", None, 0.2) + "'");
+    OmniSharp#py#eval('vim.command("let s:alive = \"%s\"" % getResponse("/checkalivestatus", None, 0.2))')
     return s:alive ==# 'true'
```

Comment:
This is getting a bit convoluted - a vim function call to execute a python command which executes a vim command. Shall we simplify it like this?
```vim
let alive = OmniSharp#py#eval('getResponse("/checkalivestatus", None, 0.2)')
return alive ==# 'true'
```